book/src/install.md

```diff
@@ -87,6 +88,16 @@ accepts the new settings on first use.
 If you are using a version of Nix without flakes enabled,
 [install Cachix CLI](https://docs.cachix.org/installation) and use
 `cachix use helix` to configure Nix to use cached outputs when possible.
+
+### AppImage
+
+Install Helix using [AppImage](https://appimage.org/).
+Download Helix AppImage from the [Releases](https://github.com/helix-editor/helix/releases) page.
```

Comment:
```suggestion
Download Helix AppImage from the [latest release](https://github.com/helix-editor/helix/releases/latest) page.
```

We can point to the latest release with that special `/latest` URL

Reply:
Ok